Resources/Prototypes/_CD/Entities/Clothing/Head/hats.yml

```diff
@@ -8,4 +8,37 @@
     sprite: _CD/Clothing/Head/Hats/boaterhat.rsi
   - type: Clothing
     sprite: _CD/Clothing/Head/Hats/boaterhat.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatAnimalHeadslimeYellow
+  name: yellow headslime hat
```

Comment:
Very minor nitpick, but you are calling it a "headslime hat" which is basically saying "hat" twice.

Reply:
Ignore me, I did not realize this was consistent with what upstream does.